packages/volto/src/components/manage/Widgets/RecurrenceWidget/RecurrenceWidget.jsx

```diff
@@ -249,7 +263,7 @@ class RecurrenceWidget extends Component {
   setRecurrenceStartEnd = () => {
     const start = this.props.formData?.start;
 
-    let _start = this.getUTCDate(start).startOf('day').toDate();
+    const _start = new Date(start); //The date is already in utc from plone, so this is not necessary: this.getUTCDate(start).startOf('day').toDate();
```

Comment:
This is more concise.

```suggestion
         // The `start` date from Plone is in UTC
        const _start = new Date(start);
```